### ISSUE TITLE
arguments to programs in GitHub Actions are incorrectly interpreted as dependencies

### ISSUE THREAD

**Issue by Zamiell** [webpro/knip#344]
This is on knip version 2.40.1.

Steps to reproduce:

```
git clone git@github.com:IsaacScript/isaacscript.git
cd isaacscript
git checkout aa5b9e68ca3ec147d0b87ec83da8bd8a536dcf61
yarn
npx knip
```

Part of the output is as follows:

```
Unlisted dependencies (4)
build                      .github/workflows/ci.yml
lint                       .github/workflows/ci.yml
test                       .github/workflows/ci.yml
```

This is a false positive. I believe that Knip is listing this because of the following line in my "ci.yml" file:

```yml
      - run: npx tsx ./scripts/getMonorepoPackageNames.ts build
```

Here, we provide a command-line positional argument to the "getMonorepoPackageNames.ts" script, to let it know that it should only give us the monorepo package names that contain a "build" script. But Knip seems to interpret this as "build" being some kind of dependency for some reason.

**Comment by webpro** [comment]
Thanks for reporting this, I think Knip can handle `tsx` just like the `node` executable and this problem should go away.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.41.0. See [Release 2.41.0](https://github.com/webpro/knip/releases/tag/2.41.0) for release notes._

**Comment by Zamiell** [comment]
Awesome, thank you webpro!
